operations/sthree.go

```diff
@@ -267,6 +268,12 @@ func s3SyncToCmd() cli.Command {
 			if err != nil {
 				return errors.Wrap(err, "problem getting new bucket")
 			}
+			syncOpts := pail.ParallelBucketOptions{
+				Workers:      runtime.NumCPU(),
```

Comment:
can we add a n argument to make this configurable? we probably want 2x this in most cases?